index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  </head>
+  <body>
+
+   <div class="app-wrap">
+     <header class="app-header">
+         <h1>Welcome</h1>
+         <figure>
+           <img src="images/Union.png" alt="union image">
+         </figure>
+     </header>
+     <div class="for_background">
+       <div class="headline">
+         <div class="primary">
+            <h4>Hey there. I'm Richard</h4>
+
+            <h4>I'm a software developer</h4>
```

Comment:
- [ ] As mentioned in the comment above, this is the perfect place to use your `<h1>` tag! I suggest the following:
```
<h1>Hey there. I'm Richard</h1>
<p>I'm a software developer</p>
```
> Note that you shouldn't have two heading tags with no other text element, such as `<p>` for instance, between them. If you want to have them in separate lines, consider using a `<br>` tag instead. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br).